packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -15,15 +15,21 @@ type TDatePickerOnChangeEvent = {
 };
 type TDatePicker = Omit<
     React.ComponentProps<typeof Native> & React.ComponentProps<typeof Input> & React.ComponentProps<typeof Calendar>,
-    'value' | 'onSelect' | 'is_datepicker_visible' | 'placement' | 'style' | 'calendar_el_ref' | 'parent_ref'
+    | 'date_format'
+    | 'onSelect'
+    | 'is_datepicker_visible'
+    | 'is_placeholder_visible'
+    | 'placement'
+    | 'style'
+    | 'calendar_el_ref'
+    | 'parent_ref'
+    | 'onChange'
+    | 'onChangeInput'
 > & {
-    mode: string;
-    start_date: moment.Moment;
+    mode?: string;
     value: moment.Moment | string;
     onChange: (e: TDatePickerOnChangeEvent) => void;
-    footer: React.ReactElement;
```

Comment:
Just curious: removed props here were omitted on line 16?

Reply:
@kate-deriv no, because it's already included in the union type of children which this DatePicker type consists of at line 17 :)

Reply:
footer is prop of datePickerCalender